queries.sql

```diff
@@ -4,11 +4,11 @@ SELECT * FROM animals; ---> to check whrther the updated item changed or not
 
 SELECT * FROM animals WHERE Name like '%mon';
 
-SELECT * FROM animals WHERE Date_of_birth >= '2016-02-25' AND Date_of_birth <= '2019-02-27';
+SELECT * FROM animals WHERE Date_of_birth >= '2016-01-01' AND Date_of_birth <= '2019-12-31';
```

Comment:
[REQUIRED] Kindly return the names of animals as mentioned in the requirements. Please refer to the following query

```ruby
SELECT name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/31/12';
```